src/test/java/org/jenkinsci/plugins/oic/MockHttpServletResponse.java

```diff
@@ -0,0 +1,199 @@
+package org.jenkinsci.plugins.oic;
+
+import jakarta.servlet.ServletOutputStream;
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Collection;
+import java.util.Locale;
+
+public class MockHttpServletResponse implements HttpServletResponse {
```

Comment:
why not just use Mockito to mock `HttpServletResponse`?

Reply:
Mainly because there is also a `MockHttpServletRequest` class. Will have a look to use Mockito